zerver/actions/realm_settings.py

```diff
@@ -409,15 +409,13 @@ def do_scrub_realm(realm: Realm, *, acting_user: Optional[UserProfile]) -> None:
     # more secure against bugs that may cause Message.realm to be incorrect for some
     # cross-realm messages to also determine the actual Recipients - to prevent
     # deletion of excessive messages.
-    all_recipient_ids_in_realm = (
-        list(Stream.objects.filter(realm=realm).values_list("recipient_id", flat=True))
-        + list(UserProfile.objects.filter(realm=realm).values_list("recipient_id", flat=True))
-        + list(
-            Subscription.objects.filter(
-                recipient__type=Recipient.HUDDLE, user_profile__realm=realm
-            ).values_list("recipient_id", flat=True)
-        )
-    )
+    all_recipient_ids_in_realm = [
+        *Stream.objects.filter(realm=realm).values_list("recipient_id", flat=True),
+        *UserProfile.objects.filter(realm=realm).values_list("recipient_id", flat=True),
+        *Subscription.objects.filter(
+            recipient__type=Recipient.HUDDLE, user_profile__realm=realm
+        ).values_list("recipient_id", flat=True),
+    ]
```

Comment:
Does this change whether the queries are lazily evaluated or not? I think the answer is they still happen at this point due to the `[]` syntax, but am not sure. (I'm not sure it would be a bad thing to change that).

Reply:
The query is evaluated at the point of interpolation into the list. There’s no way for `[]` syntax to result in anything other than a list.